css/custom.css

```diff
@@ -4,7 +4,8 @@ nav.navbar {
 }
 
 footer.navbar {
-
+    background-color: #fff;
+    border: 1px solid rgba(0,0,0,.1);
```

Comment:
Yeah actually I prefer it that way too!

I initially added the white background if the the footer was still fixed but now it's not needed indeed 😄